int32 index

bool active

geometry_msgs/Pose pose

# size_x := dimensions[0]
# size_y := dimensions[1]
# size_z := dimensions[2]
float64[] dimensions
